storage/Devices/Encryption.h

```diff
@@ -143,6 +143,24 @@ namespace storage
 	 */
 	void set_pbkdf(const std::string& pbkdf);
 
+	/**
+	 * Get the integrity. For now we only expect AEAD.
+	 *
+	 * Currently only supported for LUKS2.
```

Comment:
Please add that it is even experimental in cryptsetup/luks.

Reply:
done